tools/refguide_check.py

```diff
@@ -695,11 +694,6 @@ def check_output(self, want, got, optionflags):
         if want.lstrip().startswith("#"):
             return True
 
-        # python 2 long integers are equal to python 3 integers
-        if self.int_pattern.match(want) and self.int_pattern.match(got):
```

Comment:
I tested with `time python runtests.py --refguide` and everything passes

Reply:
I'm adding in #15306 about changes to refguide_check.py that I plan to upstream.